### PR TITLE
Add notes about dependencies management

### PR DESCRIPTION
**Pull request by adiroiban**
# Scope

We  need a few words to document how we manage the versions... how we use pypi.chevah.com and why ... and how we use our .chevah1 versionos
# Changes

A separate package was created.

We basically only have python packages, as js packages are converted into Python packages
# How to try and test the changes

reviewers: @alibotean @brunogola @atykhonov 

please check that content make sense and is in sync with what we are doing right now.

feel free to update the docs and commit the changes.

Thanks!

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

**Comment by brunogola** [comment]
fixed some typos

i think the content is good, but there is a `MAYBE` line that should be resolved before merging.

otherwise

changes-approved

File: content/dependencies.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+For the Chevah project we are not making direct use of the PyPi official index,
+but rather we use a separate PyPi index at
+`pypi.chevah.com <https://pypi.chevah.com>`_
+
+We are doing this for the following reasons:
+
+* reduce the load to the official PyPi index
+* have our development system working even when the official PyPI site is down
+* keep a copy of a package, even if the upstream developer have removed it from
```
Comment (alibotean) [inline content/dependencies.rst]: have -> has

File: content/dependencies.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Since we are a Python shop, and node-js is not supported on all our targeted
+platform, we are pushing the envelope and using python/setuptools/pip to
+manage the JS packages.
+
+All JS packages used for the projects should be *browser ready*. We don't run
+anything in node-js and we run everything in a browser... multiple browsers
+in fact.
+
+The JS packages are re-packages as Python packages and hosted into our
```
Comment (alibotean) [inline content/dependencies.rst]: re-packages -> re-packaged

File: content/dependencies.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+* `JS-PACKAGE-NAME` is the name of the project
+* `upstream.version` is the same version as the upstream project
+* `.chevahN` is the Chevah downstream repackaging version.
+
+Most of the time the repackaging version will be `.chevah1`, but in some
+cases you might need to re-package the same upstream version again (
+for example if you forgot to include certain file) and then a new version
+is required.
+
+Each package contain the following code in the top level `__init__.py` file,
```
Comment (alibotean) [inline content/dependencies.rst]: contain -> contains

**Comment by alibotean** [comment]
Just some minor typos.

changes-approved

**Comment by atykhonov** [comment]
changes-approved

**Pull request merged**